modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -240,7 +240,8 @@ def form21_10210_contact_info
       # user's own claim
       # user is a veteran
       if @form_data['claim_ownership'] == 'self' && @form_data['claimant_type'] == 'veteran'
-        [@form_data['veteran_email'], @form_data.dig('veteran_full_name', 'first')]
+        email = user&.email || @form_data['veteran_email']
```

Comment:
Could veteran_email be nil here? Is it ok if nil is returned for email?

Reply:
Yes, it is ok because [it is checked here](https://github.com/department-of-veterans-affairs/vets-api/blob/0e978818018112df05b869913b70290667690e39/modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb#L77) before firing off. If there's no email, we just `return` out.